spec/index.html

```diff
@@ -757,6 +757,12 @@ <h2>Literals</h2>
       The third element, the language tag, is treated identically as in a <a>language-tagged string</a>,
       and the fourth element, <a>base direction</a>, MUST be either `ltr` or `rtl`, which MUST be in lower case.</p>
 
+    <p>The meanings of the <a>base direction</a> values are:</p>
+    <ul>
+      <li>`ltr`: indicates that the textual value is explicitly directionally set to left-to-right text.</li>
+      <li>`rtl`: indicates that the textual value is explicitly directionally set to right-to-left text.</li>
```

Comment:
```suggestion
      <li>`ltr`: indicates that the initial text direction is set to left-to-right text.</li>
      <li>`ltr`: indicates that the initial text direction is set to right-to-left text.</li>
```

The lexical form may contain code points such as U+200E and U+200F that affect the direction while rendering. I'd avoid "textual value" because in RDF "value" relates to datatype; here RDF transports the information of initial text direction but the use of the information is when rendering: [non-normative section 3.3.1](https://www.w3.org/TR/rdf12-concepts/#section-text-direction).

Reply:
Other than the second line being for `rtl` rather than `ltr`, I think this is a good change. I'm going to just fix that inline and accept the suggestion.